src/test/coins_tests.cpp

```diff
@@ -224,21 +224,21 @@ BOOST_AUTO_TEST_CASE(chained_pours)
 
     ZCIncrementalMerkleTree tree;
 
-    CPourTx ptx1;
+    JSDescription ptx1;
```

Comment:
I believe the "p" originally stood for "pour", so maybe change this to ``jstx1``? Low priority.